templates/tree-editor/example-file/example-file-command.ts

```diff
@@ -23,8 +24,14 @@ export class NewTreeExampleFileCommandHandler implements SingleUriCommandHandler
         protected readonly fileService: FileService,
         @inject(ILogger)
         protected readonly logger: ILogger,
+        @inject(WorkspaceService)
+        protected readonly workspaceService: WorkspaceService
     ) { }
     
+    isEnabled(){
+        return this.workspaceService.opened;
+    }
```

Comment:
minor, don't mean to nitpick:

```suggestion
    isEnabled() {
        return this.workspaceService.opened;
    }
```

Reply:
no nitpicking, you are right! :-)